clang/lib/Sema/SemaBounds.cpp

```diff
@@ -622,26 +643,44 @@ namespace {
 }
 
 namespace {
-  class ReplaceVariableHelper : public TreeTransform<ReplaceVariableHelper> {
-    typedef TreeTransform<ReplaceVariableHelper> BaseTransform;
+  class ReplaceLValueHelper : public TreeTransform<ReplaceLValueHelper> {
+    typedef TreeTransform<ReplaceLValueHelper> BaseTransform;
```

Comment:
While you are at it you can consider replacing `typedef` with `using`. However, this can be done in a separate PR and won't hold up this PR.

Reply:
This `typedef` is consistent with other `TreeTransform` classes in SemaBounds.cpp. We can replace all `typedef` with `using` in a separate PR.

Reply:
Created issue #1075 to track this.